logbook/logbook.go

```diff
@@ -848,17 +847,18 @@ func (book *Book) latestSavePath(branchLog *oplog.Log) string {
 //       branch
 //       ...
 func (book Book) UserDatasetBranchesLog(ctx context.Context, datasetInitID string) (*oplog.Log, error) {
```

Comment:
My inclination is that more of these methods should move away from oplog.Log and towards something strongly typed, like what is in logbook/types.go, but this is fine for now.